app.js

```diff
@@ -1,91 +1,121 @@
+// Set up loop to loop through all the questions in questionArray.
 function guessingGameLoop(){
   for(var i = 0; i < questionArray.length; i++){
     var answer = prompt(questionArray[i]);
-
     //verifying typeof answer variable
-    if(typeof answer === "string"){
-      var inputAnswer = answer.toLowerCase();
-      console.log('lowercased');
+        // This if else statement runs whenever user provides user input.
+    if(answer % 1 == 0) {
+      var inputAnswer = parseInt(answer);
+      console.log('converted to int');
     }else{
-      var inputAnswer = answer;
-      console.log('nothing done');
+      // convert user input to lower case.
+      var inputAnswer = answer.toLowerCase();
+      console.log('lower case');
     }
     if((i === 7) && inputAnswerArray.indexOf(inputAnswer) >= 5){
       console.log('That is correct! ' + inputAnswerArray[5] + ', ' + inputAnswerArray[6] + ', ' + inputAnswerArray[7] + ', and ' + inputAnswerArray[8] + ' are the answers');
+      alert('That is correct! ' + inputAnswerArray[5] + ', ' + inputAnswerArray[6] + ', ' + inputAnswerArray[7] + ', and ' + inputAnswerArray[8] + ' are the answers');
       correctCounter++;
     }else if(i === 7 && inputAnswerArray.indexOf(inputAnswer) < 5){
       console.log('That is incorrect. Please try again.');
+      alert('That is incorrect. Please try again.');
       for(var tryGuesses = 0; tryGuesses <= 4; tryGuesses++){
         var cityGuess = prompt(questionArray[7]);
         if(inputAnswerArray.indexOf(cityGuess) < 5){
           console.log('Incorrect, please try again.');
+          alert('Incorrect, please try again.');
           incorrectCounter++;
         }else if(inputAnswerArray.indexOf(cityGuess) >= 5){
           console.log('That is correct!');
+          alert('That is correct!');
           correctCounter++;
           break;
         }else{
-
         }
       }
       console.log(inputAnswerArray[5] + ', ' + inputAnswerArray[6] + ', ' + inputAnswerArray[7] + ', and ' + inputAnswerArray[8] + ' are the answers');
+      alert(inputAnswerArray[5] + ', ' + inputAnswerArray[6] + ', ' + inputAnswerArray[7] + ', and ' + inputAnswerArray[8] + ' are the answers');
     }else if((i === 6) && inputAnswerArray.indexOf(inputAnswer) === 4){
       console.log('That is correct! ' + inputAnswerArray[4] + ' is the number!');
+      alert('That is correct! ' + inputAnswerArray[4] + ' is the number!');
       correctCounter++;
     }else if(i === 6 && inputAnswerArray.indexOf(inputAnswer) !== 4){
       if(inputAnswer > inputAnswerArray[4] || inputAnswer < inputAnswerArray[4]){
         if(inputAnswer > inputAnswerArray[4]){
           console.log('The guess is too high.');
+          alert('The guess is too high.');
           incorrectCounter++;
         }else{
           console.log('The guess is too low.');
+          alert('The guess is too low.');
           incorrectCounter++;
         }for(var guessTries = 0; guessTries <= 2; guessTries++){
```

Comment:
Not good for readability to open a for loop on the same line as a closing bracket for a prior code block.